zokrates_core/src/static_analysis/reducer/constants_reader.rs

```diff
@@ -33,6 +33,26 @@ impl<'a, 'ast, T: Field> ConstantsReader<'a, 'ast, T> {
 }
 
 impl<'a, 'ast, T: Field> Folder<'ast, T> for ConstantsReader<'a, 'ast, T> {
+    fn fold_declaration_constant(
```

Comment:
seems like this was just moved up the file? maybe the diff can be made smaller by reverting the move?

Reply:
So I have this thing in my IDE where it detects weird order of functions and tries to match the same order as it was defined in a trait, this happened here. I think it's better because then it's easier to navigate.

Reply:
But I will revert for the sake of this PR